1-js/02-first-steps/12-nullish-coalescing-operator/article.md

```diff
@@ -96,71 +96,71 @@ alert(height || 100); // 100
 alert(height ?? 100); // 0
 ```
 
-- The `height || 100` checks `height` for being a falsy value, and it really is.
-    - so the result is the second argument, `100`.
-- The `height ?? 100` checks `height` for being `null/undefined`, and it's not,
-    - so the result is `height` "as is", that is `0`.
+- Wyrażenie `height || 100` sprawdza `height` pod kątem falsy wartości, i jest ona taka.
+    - w takim razie wynikiem jest drugi argument, `100`.
+- Wyrażenie `height ?? 100` sprawdza `height` pod kątem `null/undefined`, i nie jest,
```

Comment:
To "i nie jest" jest trochę mylące, przynajmniej dla mnie 😅 . Po angielsku jakoś to zgrabniej brzmi, wydaje mi się, że po polsku musimy to zrobić trochę bardziej opisowo. Może:
"Wyrażenie `height ?? 100` sprawdza `height` pod kątem `null/undefined`, a zmienna `height` nie jest ani null ani undefined"?
Czy masz może jakąś zgrabniejszą propozycję?

Reply:
Zaproponowałem: 
>  a zmienna `height` nie jest żadną z tych wartości,